account_budget_oca/models/account_budget.py

```diff
@@ -0,0 +1,234 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models, _
+from odoo.exceptions import ValidationError
+
+from_string = fields.Datetime.from_string
+
+
+# ---------------------------------------------------------
+# Budgets
+# ---------------------------------------------------------
+class AccountBudgetPost(models.Model):
+    _name = "account.budget.post"
+    _order = "name"
+    _description = "Budgetary Position"
+
+    name = fields.Char('Name', required=True)
+    account_ids = fields.Many2many(
+        comodel_name='account.account', relation='account_budget_rel',
+        column1='budget_id', column2='account_id', string='Accounts',
+        domain=[('deprecated', '=', False)])
+    crossovered_budget_line_ids = fields.One2many(
+        comodel_name='crossovered.budget.lines',
+        inverse_name='general_budget_id', string='Budget Lines',
+        oldname='crossovered_budget_line')
```

Comment:
I don't think `oldname` is needed for a One2many field.

Reply:
Done